### PR TITLE
Update PyQt5 bindings

### PR DESCRIPTION
**Pull request by pawelkoston**
Update for #108 and #91

### CODE REVIEW

**Comment by agaida** [comment]
LGTM - but i would prefer if one with more python knowledge than me would review it too.

**Comment by DonnieWest** [comment]
I've been using these bindings locally for some time now. I can affirm they work python-side :)

File: pyqt/config.py [hunk 1]
```diff
@@ -16,13 +15,14 @@ def getEnv(self,name, default):
 		return os.environ.get(name) or default
 
 	def __init__(self):
-		qtconfig = subprocess.check_output(["/usr/lib64/qt5/bin/qmake", "-query"], universal_newlines=True)
+		qmake_bin = str(subprocess.check_output(["which", "qmake"], universal_newlines=True)).strip(' \t\n\r')
```
Comment (yan12125) [inline pyqt/config.py]: IIRC with ```universal_newlines=True```, the return value of ```check_output``` is already a ```str``` object. Is ```str()``` necessary?

File: pyqt/config.py [hunk 2]
```diff
@@ -73,6 +73,9 @@ def __init__(self):
 makefile.extra_lib_dirs.append("..")
 makefile.extra_libs = ["qtermwidget5"]
 
+#Support for C++11
+makefile.extra_cxxflags.append(' -std=c++11 ')
```
Comment (yan12125) [inline pyqt/config.py]: I guess spaces are not necessary here? For example ```makefile.extra_cxxflags.append('-std=c++11')```

**Comment by yan12125** [comment]
Well, could you do rebasing instead of merging? It's better to keep a pull request isolated (not changing unrelated files).

**Comment by pawelkoston** [comment]
These commits (	3493a91,	92aa765, 1a15d36 ) are unrelated to this pull request. I'll fix the issues that You pointed. Should I create a new pull request or try to rebase this one ?

**Comment by DonnieWest** [comment]
Sorry to bother, but any word on this? Looking forward to rebase some pull requests based on this :)

**Comment by yan12125** [comment]
Sorry for being late - I was occupying by other works

**Pull request merged**